crates/ruma-common/src/identifiers/crypto_algorithms.rs

```diff
@@ -16,6 +16,14 @@ pub enum DeviceKeyAlgorithm {
     /// The Curve25519 ECDH algorithm.
     Curve25519,
 
+    /// The Curve25519 algorithm where one key is used for both DH and signatures.
+    #[cfg(feature = "unstable-msc3983")]
+    Curve25519s,
```

Comment:
I don't see those in the MSC. Will they be added or do they come from another MSC?

Reply:
A question for @poljar 😅

Reply:
Eh right, we need a separate X3DH MSC for this, let's leave it out.